chronix-server-test-integration/src/main/java/de/qaware/chronix/solr/ChronixTestFunctions.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2016 QAware GmbH
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package de.qaware.chronix.solr;
+
+import de.qaware.chronix.converter.common.DoubleList;
+import de.qaware.chronix.converter.common.LongList;
+import de.qaware.chronix.timeseries.MetricTimeSeries;
+
+import java.util.function.BinaryOperator;
+import java.util.function.Function;
+
+/**
+ * Class with test functions for the integration test.
+ *
+ * @author f.lautenschlager
+ */
+public class ChronixTestFunctions {
```

Comment:
\[Checkstyle\] ERROR: Utility classes should not have a public or default constructor\.